provider-ui/package.json

```diff
@@ -9,7 +9,7 @@
     "test": "eslint .",
     "next": "next",
     "build": "next build",
-    "start": "NODE_ENV=production nodemon ui_dev_server.js",
+    "start": "NODE_ENV=production node ui_dev_server.js",
```

Comment:
Great to see nodemon was removed.
I think `NODE_ENV=production` shouldn't be there either.
Let's go with: `"start": "node ui_dev_server.js"` @kushthedude

Reply:
In production we use next export... I'm little curious, why NODE_ENV=production is specified though...